### PR TITLE
[JENKINS-42376] - Add executor name on Unexpected exception.

### PR DESCRIPTION
**Pull request by godfath3r**
See [JENKINS-42376](https://issues.jenkins-ci.org/browse/JENKINS-42376).

## Changelog entry:

* JENKINS-42376 - Report Executor name in the Unexpected executor death message

### CODE REVIEW

File: core/src/main/java/hudson/model/Executor.java [hunk 1]
```diff
@@ -444,7 +444,7 @@ public SubTask call() throws Exception {
             LOGGER.log(FINE, getName()+" interrupted",e);
             // die peacefully
         } catch(Exception | Error e) {
-            LOGGER.log(SEVERE, "Unexpected executor death", e);
+            LOGGER.log(SEVERE, getName()+" Unexpected executor death", e);
```
Comment (daniel-beck) [inline core/src/main/java/hudson/model/Executor.java]: Should be separated with a colon or otherwise be grammatical.

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Seems it closes JENKINS-42376, right?

**Comment by godfath3r** [comment]
Yes.

**Comment by oleg-nenashev** [comment]
thanks!

**Pull request merged**